### PR TITLE
Add query params for labels

### LINKED ISSUE #1405: Setting label as a variable not possible in Memgraph
Query parameters are able to be inserted in Memgraph for literals. That includes property values and intermediate variables. 
However, labels are hardcoded in the query and no one is currently able to set an arbitrary label via the query parameters.

### PR DESCRIPTION
**Pull request by DavIvek**
Currently, we are able to use query parameters with property values and intermediate variables. However, labels are hardcoded in the query and no one is currently able to set an arbitrary label via the query parameters.
This PR should enable users to set an label via the query parameters.
Also, this PR came out of #1448 which is closed, because approach was probably wrong.

[master < Task] PR
- [x] Check, and update documentation if necessary
- [x] Provide the full content or a guide for the final git message

To keep docs changelog up to date, one more thing to do:
- [x] Write a release note here, including added/changed clauses
- [ ] Tag someone from docs team in the comments

Release note: Besides using parameters with properties, now it's possible to use them with labels.

closes #1405

### CODE REVIEW

**Comment by DavIvek** [comment]
TODO:

- [ ] write tests

File: src/query/cypher_query_interpreter.cpp [hunk 1]
```diff
@@ -10,9 +10,12 @@
 // licenses/APL.txt.
 
 #include "query/cypher_query_interpreter.hpp"
+#include <string_view>
+#include "query/frontend/ast/ast.hpp"
```
Comment (antoniofilipovic) [inline src/query/cypher_query_interpreter.cpp:14]: Do you need these 2 includes

**Review by antoniofilipovic — COMMENTED** [review]
All looks good, just check on these 2 includes :muscle:

File: src/query/cypher_query_interpreter.cpp (hunk 1 above)
Comment (DavIvek) [reply src/query/cypher_query_interpreter.cpp]: no, I'll remove them.

**Review by antoniofilipovic — APPROVED** [review]

**Pull request merged**